### ISSUE TITLE
Add GeoJSON<->JSON conversion (and vice versa)

### ISSUE THREAD

**Issue by odscjames** [Open-Telecoms-Data/lib-cove-ofds#25]
Apparently there is script in standard repo to do this, we want this to be moved into this repository. Once that is done, make sure cove can load both, and can provide a conversion in the same way it converts to XLSX.

**Comment by odscjames** [comment]
I'm going to say this is basically done once next PR has been merged in. I've spun other issues off to allow them to be prioritised / actioned separately. Any small details could be handled in new small issues as they are discovered!

**Comment by duncandewhurst** [comment]
Closing in favour of https://github.com/Open-Telecoms-Data/cove-ofds/issues/5, which has some issues that need resolving.
